portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,12 +21,12 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/hello")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Marissa!</h1>");
```

Comment:
Clicking the button "Fetch a Hello!" button currently returns "\<h1>Hello Marissa!\</h1>". I think you're missing a "/"

Reply:
I'm going leave it as a TODO and just remove the tags. I spent too much time trying to resolve the issue, so I'll come back to this later.